tck/features/ReturnAcceptance.feature

```diff
@@ -98,17 +189,28 @@ Feature: ReturnAcceptanceTest
 
   Scenario: Support column renaming
     Given an empty graph
-      And having executed: CREATE (:Singleton)
-    When executing query: MATCH (a) WHERE id(a) = 0 RETURN a AS ColumnName
+    And having executed: CREATE (:Singleton)
+    When executing query:
+      """
+      MATCH (a)
+      WHERE id(a) = 0
+      RETURN a AS ColumnName
```

Comment:
I am not so sure about this one... people may want the freedom to choose their own alias names, in order that it conforms with their application, for example.